.evergreen/run-gssapi-auth-test.sh

```diff
@@ -41,4 +41,4 @@ echo "Running tests with ${JDK}"
 -Dorg.mongodb.test.uri=${MONGODB_URI} \
 -Pgssapi.enabled=true -Psun.security.krb5.debug=true -Pauth.login.config=file://${PROJECT_DIRECTORY}/.evergreen/java.login.drivers.config \
 -Pkrb5.kdc=${KDC} -Pkrb5.realm=${REALM} -Psun.security.krb5.debug=true \
-driver-core:test --tests GSSAPIAuthenticationSpecification
+driver-core:test --tests GSSAPIAuthenticationSpecification --tests GSSAPIAuthenticatorSpecification --tests KerberosSubjectProviderTest
```

Comment:
Almost forgot to do this...  But this is the only way to get the new tests to run